tests/unit_test/callback/pyscript_handler_test.py

```
@@ -2065,4 +2057,628 @@
+    cfg = MagicMock()
+    cfg.to_mongo.return_value.to_dict.return_value = {
+        "smtp_url": "smtp.gmail.com",
+        "smtp_port": 587,
+        "tls": True,
+        "smtp_userid": {"value": "user@example.com"},
+        "smtp_password": {"value": "password123"}
+    }
+    return cfg
+
+@responses.activate
+def test_send_email_direct(smtp_config):
```

Ruff (0.8.2)

Redefinition of unused `test_send_email_direct` from line 1287

(F811)